stepper_test/stepper_test.ino

```diff
@@ -1,85 +1,89 @@
-
-#define BTN_UP_PIN 13
-#define BTN_DOWN_PIN 12
 #define UP HIGH
 #define DOWN LOW
 #define TOP 0
 #define BOTTOM 1
+#define baudRate 9600
```

Comment:
The widely accepted convention is to have capitalized macros.

Here is some general info about them and what you can do with them.

https://www.programiz.com/c-programming/c-preprocessor-macros